tests/__snapshots__/test_mealie.ambr

```diff
@@ -259,16 +259,16 @@
       'group_id': '0bf60b2e-ca89-42a9-94d4-8f67ca72b157',
       'household_id': 'cd2bb87f-5e4c-4dc6-8477-af9537200014',
       'mealplan_date': datetime.date(2024, 1, 21),
-      'mealplan_id': 207,
+      'mealplan_id': '207',
       'recipe': dict({
         'description': '',
         'group_id': '0bf60b2e-ca89-42a9-94d4-8f67ca72b157',
         'household_id': 'cd2bb87f-5e4c-4dc6-8477-af9537200014',
-        'image': None,
+        'image': 'None',
```

Comment:
Why is this a string None now?

Reply:
From what I can gather from the HA changes Mashumaro now type checks stronger and we are not allowing None types on most of the fields, because we don't care about them.
I know the shopping list fields are manipulated so why I've added str | None, you may need to do that to recipes but not sure of impact.